gbpservice/nfp/configurator/drivers/firewall/vyos/vyos_fw_driver.py

```diff
@@ -364,7 +364,8 @@ def configure_routes(self, context, resource_data):
                "primary service at: %r" % mgmt_ip)
         LOG.info(msg)
         try:
-            resp = requests.post(url, data=data, timeout=self.timeout)
+            routes_timeout = 60
```

Comment:
There is a big issue w.r.t timeouts.
Observed that by default timeout is 30 secs, This particular rest call without sharing took 27~29 secs, with sharing taking 34~36 secs, hence failing.
For now increased timeout to large value 60 secs, but we need to validate all timeouts properly.